recipes/sextractor/0003-Rename_CLI_tool.patch

```diff
@@ -0,0 +1,189 @@
+diff --git a/man/sex.1.in b/man/source-extractor.1.in
+similarity index 84%
+rename from man/sex.1.in
+rename to man/source-extractor.1.in
+index d0f7208..b3e2a64 100644
+--- a/man/sex.1.in
++++ b/man/source-extractor.1.in
+@@ -1,18 +1,18 @@
+ .TH SEXTRACTOR "1" "@DATE3@" "SExtractor @PACKAGE_VERSION@" "User Commands"
+ .SH NAME
+-sex \- extract a source catalogue from an astronomical FITS image
++source-extractor \- extract a source catalogue from an astronomical FITS image
+ .SH SYNOPSIS
+-.B sex \fIimage\fR [\fI-c configuration-file\fR]
++.B source-extractor \fIimage\fR [\fI-c configuration-file\fR]
+ .RS
+ [\fI-parameter1 value1 -parameter2 value2 ...\fR]
+ .RE
+ .TP
+-.B sex \fIimage\_detect\fR \fIimage_measure\fR [\fI-c configuration-file\fR]
++.B source-extractor \fIimage\_detect\fR \fIimage_measure\fR [\fI-c configuration-file\fR]
+ .RS
+ [\fI-parameter1 value1 -parameter2 value2 ...\fR]
+ .RE
+ .TP
+-.B sex \fI-d\fR
++.B source-extractor \fI-d\fR
+ .SH DESCRIPTION
+ SExtractor is a program that builds a catalogue of objects from an astronomical
+ image. Although it is particularly oriented towards reduction of large scale
+diff --git a/man/sex.x b/man/source-extractor.x
+similarity index 8%
+rename from man/sex.x
+rename to man/source-extractor.x
+index c6f97ad..134310f 100644
+--- a/man/sex.x
++++ b/man/source-extractor.x
+@@ -1,2 +1,2 @@
+ [name]
+-sex \- extract a source catalog from an astronomical FITS image
++source-extractor \- extract a source catalog from an astronomical FITS image
+diff --git a/configure.ac b/configure.ac
+index 7ff6cce..6b6415d 100644
+--- a/configure.ac
++++ b/configure.ac
+@@ -98,6 +98,7 @@ if test x$enable_auto_flags = xyes; then
+ fi
+ LT_INIT
+ AC_PROG_INSTALL
++AC_PROG_LN_S
+ 
+ # Checks for libraries.
+ AC_CHECK_LIB(m, sin)
+@@ -335,7 +336,7 @@ debian/changelog
+ doc/Makefile
+ doc/src/conf.py
+ man/Makefile
+-man/sex.1
++man/source-extractor.1
+ src/Makefile
+ src/fits/Makefile
+ src/levmar/Makefile
+diff --git a/doc/src/Config.rst b/doc/src/Config.rst
+index be953db..111823d 100644
+--- a/doc/src/Config.rst
++++ b/doc/src/Config.rst
+@@ -19,7 +19,7 @@ By redirecting the standard output of SExtractor to a file, one creates a config
+ 
+ .. code-block:: console
+ 
+-  $ sex -d > default.sex
++  $ source-extractor -d > default.sex
```

Comment:
IMHO we don't. Its not uncommon for file extension to be abbreviated to three letters, and I don't want to have to dig into the internals of the C code.

This is where the buck stop for me.

Reply:
I don't agree here. I doubt anyone would pick this file extension today. `.cfg` actually makes more sense.

Reply:
I agree that the extension should change, but I don't think that is the kind of thing we can reasonably do in a patch.  The extension is an API and so breaking that would break the users of the package.  If we want this fixed we'd actually have to fork the package fully. (Which I think we might still want to do if they don't accept these changesets upstream.)

Reply:
I actually don't think the extension is an API. The code reads any text file with the right format IIRC.